### PR TITLE
GROW-674 Borrower loan description image

### PR DESCRIPTION
**Pull request by emuvente**

### CODE REVIEW

File: src/components/BorrowerProfile/BorrowerImage.vue [hunk 1]
```diff
@@ -0,0 +1,136 @@
+<template>
+	<picture
+		class="
+			tw-inline-block
+			tw-relative
+			tw-overflow-hidden
+		"
+		:style="`padding-bottom: ${aspectRatio * 100}%;`"
```
Comment (mcstover) [inline src/components/BorrowerProfile/BorrowerImage.vue:8]: Does this manifest as actual padding below the image?

**Review by mcstover — APPROVED** [review]

File: src/components/BorrowerProfile/BorrowerImage.vue (hunk 1 above)
Comment (mcstover) [reply src/components/BorrowerProfile/BorrowerImage.vue:8]: Seems like more of an offset technique. I was just wondering about reusability on something like the loan card...

**Review by mlibazisi — APPROVED** [review]
👍

File: src/components/BorrowerProfile/BorrowerImage.vue (hunk 1 above)
Comment (emuvente) [reply src/components/BorrowerProfile/BorrowerImage.vue:8]: Yeah, it's something we've used on loan cards before (see https://github.com/kiva/ui/blob/master/src/components/LoanCards/LoanCardImage.vue#L112-L117). The element with the padding is position relative, and the image it contains is position absolute. The padding value itself is the aspect ratio of the image, as a percent, and what that gives you is a responsive image placeholder (for while the image is loading) that automatically matches the expected height of the image no matter what width it is displayed at.

File: src/components/BorrowerProfile/BorrowerImage.vue [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+		<img
+			v-if="hash"
+			class="
+				tw-absolute
+				tw-w-full
+				tw-h-full
+				tw-object-contain
+			"
+			:src="defaultUrl"
+			:alt="alt"
```
Comment (ryan-ludwig) [inline src/components/BorrowerProfile/BorrowerImage.vue:20]: a `loading="lazy"` attribute or `:loading="loading"` and using a prop would be good here.

**Review by ryan-ludwig — APPROVED** [review]
Really nice work!

**Pull request merged**